src/Formatter/ClassNameFormatter.php

```diff
@@ -28,15 +28,26 @@ public function commandHandled($command)
 
     /**
      * @param object $command
-     * @param Exception $e
      * @return string|null
      */
-    public function commandFailed($command, Exception $e)
+    public function commandFailed($command)
     {
-        $commandClass = get_class($command);
-        $exceptionClass = get_class($e);
-        $exceptionMessage = $e->getMessage();
+        return 'Command failed: ' . get_class($command);
+    }
 
-        return "Command failed: {$commandClass} threw the exception {$exceptionClass} ({$exceptionMessage})";
+    /**
+     * {@inheritDoc}
+     */
+    public function commandContext($command)
+    {
+        return ['class' => get_class($command)];
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function completeCommandContextOnFailure(array $currentContext, \Exception $e)
+    {
+        return ['error' => ['class' => get_class($e), 'message' => $e->getMessage()]] + $currentContext;
```

Comment:
Yeah, I suppose this is the best way to do it for now (though still perhaps with the rename)